### DISCUSSION TITLE
Copy attachment files from an item to the clipboard

### DISCUSSION THREAD

**Discussion by mjthoraval** [windingwind/zotero-actions-tags#190] [Ideas]
I would like to get a right-click menu option to copy the attachment files  from an item to the cliboard.

I sometimes need to share a paper with multiple attachments by email. For example, with 10 videos.
In the current situation, I need to use the `Show File` option for each file individually and then add them to the email, which is quite painful.

**Comment by redleafnew** [comment]
https://github.com/windingwind/zotero-actions-tags/discussions/195
copy the first attachment when selected an item to the clipboard.
And you could copy the attachments one by one by select each attachment.
